docs/en/toolkit/learning-resources/guides/dynamic_filesystem_configuration.md

```diff
@@ -0,0 +1,398 @@
+
+**Step 6:** Assign the Model task on the **filet** asset to yourself, so you can find it easily for testing purposes. 
+
+### Setting up the schema
+
+You’ve now enabled a Set custom entity, created a Set entity called “Dining Room”, and linked an Asset entity to the Dining Room set. You’ve got all the pieces in place in your Shotgun site to now modify your folder structure. When an artist starts working on a task, Toolkit uses the associated Shotgun data to determine what folders to create in the filesystem. New folders are created and named automatically based on the pipeline configuration’s schema. 
+
+Now it’s time to define the folder structure you want Toolkit to dynamically generate as artists step through the production pipeline. This is done by editing the schema.
+
+{% include info title="Note" content="It’s a good practice to test configurations in a *cloned* configuration, so as not to affect the active production configuration. The cloning process creates a *copy* of the configuration that you can edit safely before pushing changes to the live config. You can find details on cloning your config in the [Configuration Staging and Rollout](https://support.shotgunsoftware.com/hc/en-us/articles/219033168-Configuration-staging-and-rollout#Cloning%20your%20Configuration) document." %}
+
+**Step 7:** Navigate to your pipeline configuration. Drill down to the schema folder, `<pipeline_configuration_root>/config/core/schema` and open the `project` folder.
+
+![Python app](../../../../images/toolkit/learning-resources/guides/dynamic_filesystem_configuration/15_file_structure.png)
+
+The current schema, which looks like this:
+
+`<project>/assets/<asset_type>/<asset>/<step>`
+
+supports dynamically creating this folder structure:
+
+`the_other_side/assets/prop/filet/model`
+
+We want the following structure instead:
+
+`the_other_side/assets/Dining-Room/Prop/filet/model`
+
+To achieve this, you would set up the schema like this:
+
+`<project>/assets/<CustomEntity01>/<asset_type>/<asset>/<step>`
+
+The Set entity is represented as `CustomEntity01`. While we gave CustomEntity01 the *display name* of Set in Shotgun, in our configuration, we’ll always refer to it by its system name, `CustomEntity01`. 
+
+### How the schema uses YAML files
+
+A schema can contain static and dynamic folders. If you have a static folder in your schema called “assets”, it will correspond to a single folder in your production filesystem called “assets”. On the other hand, you might have a dynamic folder in your schema named “asset” that represents one folder for each asset in your project. Every dynamic folder has a corresponding YAML file next to it with the same name as the folder (eg, `asset/` and `asset.yml`) that defines the rules for how to generate production folders based on the schema folder.
+
+ ### Create the new folder and YAML file for the Set entity
```

Comment:
This isn't getting parsed as a heading by kramdown, I think you just need to remove the leading space.

Reply:
good catch. fixed.